submit.py

```diff
@@ -77,6 +80,24 @@ def get_resource_limit_scale(self, key, sec="SubmitFile"):
     def cleanup(self, cmd, direc):
         pass
 
+    def get_presigned_put_url(self):
```

Comment:
I think this function could also be factored out into a separate file.  Maybe put it with the minio function from client.py that I suggested to factor out?

Reply:
@dsschult I moved this to client_util.py as well.